- Check for a passivetex format file and tell the user, if it's missing.
- Modified to use texmf-site directory by ssuominen at gentoo.org.
- http://bugs.debian.org/190840

diff -ur xmlto-0.0.22.orig/format/fo/dvi xmlto-0.0.22/format/fo/dvi
--- xmlto-0.0.22.orig/format/fo/dvi	2009-03-20 10:28:02.000000000 +0200
+++ xmlto-0.0.22/format/fo/dvi	2009-05-22 15:20:33.000000000 +0300
@@ -11,6 +11,10 @@
     echo >&2 "Can't process, xmltex tool not found at $XMLTEX_PATH."
     exit 3
   fi
+  if [ ! -f /usr/share/texmf-site/tex/xmltex/passivetex/fotex.xmt ]; then
+    printf "PassiveTeX is needed for this format, but it is not installed. Please install\nthe passivetex package.\n"
+    exit 1
+  fi
   # Work around stupid tetex bug with '_' in filenames
   # Also work around stupid tetex limitation with long lines (bug #101055)
   sed -e "s,/>,\n/>,g" "$XSLT_PROCESSED" >tmp.fo
diff -ur xmlto-0.0.22.orig/format/fo/pdf xmlto-0.0.22/format/fo/pdf
--- xmlto-0.0.22.orig/format/fo/pdf	2009-03-20 10:28:33.000000000 +0200
+++ xmlto-0.0.22/format/fo/pdf	2009-05-22 15:20:33.000000000 +0300
@@ -13,6 +13,10 @@
 	    echo >&2 "Can't process, pdfxmltex tool not found at $PDFXMLTEX_PATH."
 	    exit 3
 	  fi
+	  if [ ! -f /usr/share/texmf-site/tex/xmltex/passivetex/fotex.xmt ]; then
+	    printf "PassiveTeX is needed for this format, but it is not installed. Please install\nthe passivetex package.\n"
+	    exit 1
+	  fi
 	  # Work around stupid tetex bug with '_' in filenames
 	  # Also work around stupid tetex limitation with long lines (bug #101055)
 	  sed -e "s,/>,\n/>,g" "$XSLT_PROCESSED" >tmp.fo
diff -ur xmlto-0.0.22.orig/format/fo/ps xmlto-0.0.22/format/fo/ps
--- xmlto-0.0.22.orig/format/fo/ps	2009-03-17 18:39:52.000000000 +0200
+++ xmlto-0.0.22/format/fo/ps	2009-05-22 15:20:33.000000000 +0300
@@ -8,6 +8,10 @@
 	  then
 	    echo >&2 "Post-process XSL-FO to DVI"
 	  fi
+	  if [ ! -f /usr/share/texmf-site/tex/xmltex/passivetex/fotex.xmt ]; then
+	    printf "PassiveTeX is needed for this format, but it is not installed. Please install\nthe passivetex package.\n"
+	    exit 1
+	  fi
 	  # Work around stupid tetex bug with '_' in filenames
 	  # Also work around stupid tetex limitation with long lines (bug #101055)
 	  sed -e "s,/>,\n/>,g" "$XSLT_PROCESSED" >tmp.fo
